### PR TITLE
Get CI tests workings

### LINKED ISSUE #324: FYI - Travis CI Failing
To anyone looking to contribute, the current Travis CI is failing because it has not been fully setup.  Currently the `cbpro/` filepath is not a visible package to the tests, so a fix for this will be added soon to comply with PEP420.

### PR DESCRIPTION
**Pull request by alimcmaster1**
Fixes #324 

CI is now working. So peoples PRs arnt blocked by failing CI.
We may revisit implementation here. But this should be okay for now :)

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -4,3 +4,5 @@ six==1.10.0
 websocket-client==0.40.0
 pymongo==3.5.1
 pytest>=3.3.0
+pytest>=3.3.0
```
Comment (yiwensong) [inline requirements-dev.txt]: this looks like a duplicate from the previous line.

File: tests/test_authenticated_client.py [hunk 2]
```diff
@@ -23,6 +23,7 @@ def test_place_order_input_2(self, dc):
             r = dc.place_order('BTC-USD', 'buy', 'limit',
                                cancel_after='123', tif='ABC')
```
Comment (danpaquin) [inline tests/test_authenticated_client.py]: the tif param needs to be renamed, 'time-in-force'

Comment (alimcmaster1) [reply tests/test_authenticated_client.py]: Ahh yep just seen #317

**Comment by alimcmaster1** [comment]
Thanks both updated as per your comments.

**Comment by alimcmaster1** [comment]
Thanks @danpaquin

**Pull request merged**